packages/shared/src/buttons/CloseButton.js

```diff
@@ -13,7 +13,7 @@ type Props = {|
 
 export default function CloseButton({ onPress }: Props) {
   return (
-    <Button onPress={onPress} type="secondary">
```

Comment:
I don't like web APIs in React Native 😄 `onPress` feels more platform agnostic

Reply:
It feels weird to me to add onClick here but, seems to be requirement from orbit that the API is the same as for the web components 🙈